src/stdlib/number.md

```diff
@@ -118,4 +118,4 @@ The name `Number` below stands for one of the wrappers `F32` or `F64` representi
 * ```ts
   function toString(radix?: i32): string
   ```
-  Returns the respective basic value converted to a string.
+  Returns the respective basic value converted to a string. The `radix` parameter ignored here.
```

Comment:
Makes me wonder, do we already have the infra to support hex floats, in theory?

```
> Math.PI.toString(16)
'3.243f6a8885a3'
```

Reply:
I guess we could support hexadecimal float format which supported in C++ for example. But hex floats in JS is quite unuseful